modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -26,6 +26,54 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the installation directory that
 contains the `install-config.yaml` file you created.
 
+. Set the default Maximum Transmission Unit (MTU) according to the network plugin:
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300, see [AWS Documentation](https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html). 
+The Cluster network MTU must be always less than the EC2 MTU to account for the cluster network overlay overhead. The specific overhead is determined by your network plugin:
+- OVN-Kubernetes: 100 bytes
+- OpenShift SDN: 50 bytes
+
+Check the ['MTU value selection' for more details](https://docs.openshift.com/container-platform/4.12/networking/changing-cluster-network-mtu.html#mtu-value-selection_changing-cluster-network-mtu)
```

Comment:
Remove this line, and put these lines in the assembly after the include statement for this module ( https://github.com/openshift/openshift-docs/blame/main/installing/installing_aws/installing-aws-localzone.adoc#L80):

```
[role="_additional-resources"]
.Additional resources

* xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[MTU value selection]
```

Reply:
👍🏽